reading/structs.livemd

```diff
@@ -181,11 +182,11 @@ Person.greet(person)
 
 * A `Hero` will have a `:name`, `:catchphrase`, and `:secret_identity`
 
-* create a `reveal/1` function which takes in an instance of a hero and returns
+* Create a `reveal/1` function which takes in an instance of a hero and returns
 
   `"I am secretly #{hero.secret_identity}"`
 
-* create an `introduce/2` function which takes in an instance of a hero and a greeting
+* Create an `introduce/2` function which takes in an instance of a hero and a greeting
```

Comment:
I think it would be potentially worthwhile to write a script or something that finds bullet points and capitalizes the next word (if it's followed by a character). I don't think I've been very consistent about this 🤔

Reply:
Something like https://github.com/DockYard-Academy/beta_curriculum/issues/104?

Reply:
yup! We've done some work to make a similar script inserts markdown breaks before headings. There are some bugs to work out though so likely it can also handle some of these consistency issues.